src/index.html

```diff
@@ -1,15 +1,167 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div>
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+          >
+        </div>
+        <div>
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            id="name"
+            name="name"
+          >
+        </div>
+        <div>
+          <label for="age">How old are you?</label>
+          <input
+            type="number"
+            id="age"
+            name="age"
+            value ="12"
+            min="1"
+            max="150"
+            minlength="12"
+          >
+        </div>
+        <div>
+          <label for="date">Full date of birth:</label>
+          <input
+            type="date"
+            id="date"
+            name="date"
+          >
+        </div>
+        <div>
+          <label for="accept">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            id="accept"
+            name="accept"
```

Comment:
```suggestion
            name="agreement-terms"
```
this is much better, don't you think?